src/Sarif/GitInformation.cs

```diff
@@ -1,180 +0,0 @@
-﻿// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Threading;
-
-namespace Microsoft.CodeAnalysis.Sarif
-{
-    internal class GitInformation : IDisposable
```

Comment:
>GitInformation [](start = 19, length = 14)

Renamed to `GitHelper` and edited so extensively that Git can't tell it's the same file. A pity -- it held out until iteration 10.